components/scripts/lib/gradle.sh

```diff
@@ -10,20 +10,17 @@ invoke_gradle() {
     cd "${project_dir}" > /dev/null 2>&1 || die "ERROR: The subdirectory ${project_dir} (set with --project-dir) does not exist in ${project_name}." "${INVALID_INPUT}"
   fi
 
+  args+=(--init-script "${INIT_SCRIPTS_DIR}/configure-gradle-enterprise.gradle")
+
   if [ "$enable_ge" == "on" ]; then
-    args+=(--init-script "${INIT_SCRIPTS_DIR}/enable-gradle-enterprise.gradle")
+    args+=("-Dcom.gradle.enterprise.build_validation.enableGradleEnterprise=true")
```

Comment:
0: I'm curious, why are we setting these as system properties instead of as project properties?

Reply:
That's a great question. I am not sure. I only continued the pattern started by @bigdaz in #121. 

Can you think of any reason not to use a project property?

Reply:
It was my understanding that project properties aren't available when an init script evaluates. But I could well be mistaken.

Reply:
That makes sense @bigdaz. I'm having trouble finding it anywhere in the documentation, but I did make a small sample project to test.

```
* What went wrong:
A problem occurred evaluating initialization script.
> The default project is not yet available for build.
```

![image](https://user-images.githubusercontent.com/5797900/220462475-78d295d2-1712-40e2-8018-08eb49b88ef9.png)